README.md

```diff
@@ -26,6 +27,7 @@ Best way to start with ESP-BSP is trying one of the [examples](examples) on your
 
 | Example | Supported boards |
 |---|---|
+| [audio](examples/audio) | ESP32-S3-Korvo-2 |
```

Comment:
Not related to this PR, but could you remove the `rainmaker_example` from the list? We removed it from esp-bsp a while ago